packages/client/src/v2-events/utils/imageUtils.test.ts

```diff
@@ -0,0 +1,108 @@
+/*
```

Comment:
Same, let's rely on the ones on the legacy side for now. A lot of these tests feel like they are not very useful